Sources/AblyChat/Presence.swift

```diff
@@ -4,8 +4,8 @@ import Ably
 public typealias PresenceData = any Sendable
 
 public protocol Presence: AnyObject, Sendable, EmitsDiscontinuities {
-    func get() async throws -> any PaginatedResult<[PresenceMember]>
-    func get(params: ARTRealtimePresenceQuery?) async throws -> any PaginatedResult<[PresenceMember]>
+    func get() async throws -> any PaginatedResult<PresenceMember>
```

Comment:
You're right that it's a mistake, but there actually shouldn't be a `PaginatedResult` at all; see https://github.com/ably/ably-chat-js/blob/0b4b0f8c35d1fb789bcc1fe779c920140f4b598c/src/core/presence.ts#L123-L128. Should just be `[PresenceMember]`. Fixed in https://github.com/ably-labs/ably-chat-swift/pull/41.

Reply:
Now I wonder why it's not paginated. What if there are hundreds of users in the room?

Reply:
A fair question, but it's the same in the core SDKs, right?

Reply:
Very strange it's not paginated there, I thought it is!